### ISSUE TITLE
Arrow rendering strangely.

### ISSUE THREAD

**Issue by jordwalke** [fabrizioschiavi/pragmatapro#41]
I believe the newest pre-release renders single `->` arrows strangely. I don't recall the vertical leg of the triangle being thinner than the other portions of the triangle. Is this unintentional? It makes the arrows feel a little less "sturdy".
<img width="28" alt="screen shot 2016-09-18 at 11 15 05 pm" src="https://cloud.githubusercontent.com/assets/977348/18623764/f59517ac-7df5-11e6-8879-4935503201e7.png">

**Comment by fabrizioschiavi** [comment]
Yes, it’s intentional. I think that with this discordant design of the outline everyone can remember most easily the origins of the ligature `-` + `>` and to avoid confusion with other Unicode arrows

**Comment by jordwalke** [comment]
I'm not exactly a fan, but I use `=>` anyways, so it might not effect me much.
